label_options/caebike-label-options.json

```diff
@@ -0,0 +1,85 @@
+{
+  "MODE": [
+    {"value":"walk", "baseMode":"WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}}, "kgCo2PerKm": 0},
+    {"value":"auto_driver", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.22031},
+    {"value":"auto_passenger", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.110155},
+    {"value":"e-bike", "baseMode":"E_BIKE", "met": {"ALL": {"range": [0, -1], "mets": 5}}, "kgCo2PerKm": 0.00728},
+    {"value":"bike", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 6}}, "kgCo2PerKm": 0},
+    {"value":"taxi", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
+    {"value":"bike_scooter_share", "baseMode":"BICYCLING_SCOOTER_SHARE", "met": {"ALL": {"range": [0, -1], "mets": 3.5}}, "kgCo2PerKm": 0.00447},
```

Comment:
"BICYCLING_SCOOTER_SHARE" is not a base mode we have available - I'd suggest picking one from - {BICYCLING, E_BIKE, E_SCOOTER} which are supported by the phone code

Reply:
Hi @Abby-Wheelis, so the baseMode field only controls the icon and color shown in the trip cards....it does not affect the kgCO2perkm and MET values, correct?

Reply:
that's correct, the values you provided will be used - the base modes affect the visual representation of the modes

Reply:
Okay...thanks @Abby-Wheelis. For "BICYCLING_SCOOTER_SHARE", let's have BICYCLING as the base mode. For "PUBLIC_TRANSIT", BUS can be the base mode. For "OTHER_MODE", OTHER can be the base mode. I have suggested the most general base made option from the available choices. Please let me know if you have any questions.

Reply:
@si-agarwal That sounds good! Are you able to make those edits to this file or did you want me to edit the file?

Reply:
@Abby-Wheelis, I just made those edits in the json file. Please let me know if you have any questions.